### ISSUE TITLE
Deprecation notice with PHP 8.1.5

### ISSUE THREAD

**Issue by givanov2** [smi2/phpClickHouse#169]
Hello, thank you for the excellent library.

FYI, when using `ClickHouseDB\Client::write` with PHP 8.1.5, the following notice appears:

```
Deprecated: Return type of ClickHouseDB\Statement::current() should either be compatible with Iterator::current(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /.../vendor/smi2/phpclickhouse/src/Statement.php on line 581
```

PHP information:

```
PHP 8.1.5 (cli) (built: Apr 18 2022 23:52:55) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.5, Copyright (c) Zend Technologies
```

**Comment by isublimity** [comment]
> Fixed here #170

Thx! Release in 1.4.6

**Comment by givanov2** [comment]
Hello. Any chance this change gets in a release?
